common/test/acceptance/pages/lms/progress.py

```diff
@@ -67,6 +67,13 @@ def scores(self, chapter, section):
         # Retrieve the scores for the section
         return self._section_scores(chapter_index, section_index)
 
+    def detail_graph_contains(self, text):
+        """
+        Return a list of names of the assignments
+        that appear on the grade detail graph.
```

Comment:
Updated and renamed the function too, as it's more generic than what the above name suggests.